go.mod

```diff
@@ -8,25 +8,25 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.20
 	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/golang/mock v1.6.0
-	github.com/google/go-cmp v0.5.8
+	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/mitchellh/mapstructure v1.4.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
-	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
+	github.com/onsi/gomega v1.27.4
+	github.com/openshift/api v0.0.0-20230802084312-ab49804a4103
 	github.com/openshift/machine-api-operator v0.2.1-0.20220922080915-f7f326323f37
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.4.0
-	golang.org/x/crypto v0.0.0-20220331220935-ae2d96664a29
+	golang.org/x/crypto v0.11.0
 
 	// kube 1.25
-	k8s.io/api v0.25.1
-	k8s.io/apimachinery v0.25.1
+	k8s.io/api v0.27.1
```

Comment:
since this PR is for 4.12, do we want to bump the kubernetes version?

Reply:
We should probably not update the deps where possible

Reply:
I tried to avoid updating k8s deps at all, but we need to update the openshift api package to get the field needed in this update, which results in updating the k8s deps, which then result in errors and would need to bump to 1.27 for all of the other packages as well.

Any idea how I could get avoid this?

Reply:
Final destination of this backport is 4.11 btw. I don't think we could backport without k8s bump as oc/api depends on it?

Reply:
If there's an API change involved here I don't think we are backporting the correct work. We need to make sure that the NIC is tagged with the cluster owned tag, but it doesn't need to have access to the user defined tags from the infrastructure object, we shouldn't backport that new feature

Reply:
You are probably right with that. I will try to test this with a cluster once it gets up.